source/copilot-schema.txt

```diff
@@ -0,0 +1,63 @@
+.. _vsce-copilot-schema:
+
+===============
+/schema Command
+===============
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+
+The ``/schema`` command provides information about your collection
+schema. The |copilot| accesses your connected MongoDB deployment to
+provide schema information in the Copilot generated response. 
+
+Examples
+--------
+
+Provide a Sample Document
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Consider the ``users`` collection in the `Mflix Sample Database
+<https://www.mongodb.com/docs/atlas/sample-data/sample-mflix/#sample_mflix.users>`__.
+If you are connected to a MongoDB deployment that contains this
+collection, you can ask the |copilot| to provide a sample document:
+
+.. code-block:: none
+   :copyable: false
+
+   @MongoDB /schema Can you show a sample document from the
+   sample_mflix.users collection?
+
+GitHub Copilot uses the |copilot| to generate the following response:
+
+.. code-block:: none
+   :copyable: false
+
+   Sure, here is a sample document from the sample_mflix.users
+   collection:
+   
+   {
```

Comment:
I checked the Atlas samples page, and it shows the sample doc as:

{
--
"_id": {
"$oid": "59b99db4cfa9a34dcd7885b6"
},
"name": "Ned Stark",
"email": "sean_bean@gameofthron.es",
"password": "$2b$12$UREFwsRUoyF0CRqGNK0LzO0HM/jLhgUCNNIJ9RJAqMUQ74crlJ1Vu"
}

Reply:
I used this document to illustrate the collection schema!